### PR TITLE
DOC-1004: 5.10.2 release notes

### PR DESCRIPTION
**Pull request by tylerkelly13**
Related Ticket: DOC-1004

Description of Changes:
* 5.10.2 Release notes and other enterprise release prep.

Pre-checks:
- [x] Branch prefixed with `feature/` or `hotfix/`
- [x] `_data/nav.yml` has been updated (if applicable)
- [x] Files has been included where required (if applicable)
- [x] Files removed have been deleted, not just excluded from the build (if applicable)
- [x] (New product features only) Release Note added

Review:
- [x] Documentation Team Lead has reviewed
- [x] Product Manager has reviewed

### CODE REVIEW

**Review by ashfordneil — APPROVED** [review]

File: release-notes/release-notes5102.md [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+- [](#)
+
+###
+
+Issue
+: This issue affects ....
+
+Workaround
+: There was no known workaround at the time of the release.
+
```
Comment (lnewson) [inline release-notes/release-notes5102.md:119]:
```suggestion
```
Just putting this here so we don't forget as I would hope we have nothing to add here.

**Review by lnewson — APPROVED** [review]

File: _data/nav.yml [hunk 2]
```diff
@@ -967,16 +967,25 @@
...
   pages:
   - url: "6.0-upcoming-changes"
+  - url: "release-notes5102"
+    pages:
+    - url: "#Overview"
+    - url: "#Accompanying Premium Plugin changes"
+    - url: "#Accompanying Premium self-hosted server-side component changes"
+    - url: "#General bug fixes"
+    - url: "#Security fixes"
+    - url: "#Known issues"
```
Comment (tylerkelly13) [inline _data/nav.yml:977]:
```suggestion
```

File: release-notes/release-notes5102.md [hunk 3]
```diff
@@ -0,0 +1,124 @@
...
+## Overview
+
+{{site.productname}} 5.10.2 was released for {{site.enterpriseversion}} and {{site.cloudname}} on Monday, November 22<sup>nd</sup>, 2021. It includes {{site.productname}} 5.10.2 and additional changes to premium plugins. These release notes provide an overview of the changes for {{site.productname}} 5.10.2, including:
+
+- [Accompanying Premium Plugin changes](#accompanyingpremiumpluginchanges)
+- [Accompanying Premium self-hosted server-side component changes](#accompanyingpremiumself-hostedserver-sidecomponentchanges)
+- [General bug fixes](#generalbugfixes)
+- [Security fixes](#securityfixes)
+- [Known issues](#knownissues)
```
Comment (tylerkelly13) [inline release-notes/release-notes5102.md:17]:
```suggestion
```

**Review by metricjs — APPROVED** [review]

**Comment by tylerkelly13** [comment]
Note: @metricjs is acting PM at this time.

**Pull request merged**